### DISCUSSION TITLE
Dashy on Unraid - anyone?

### DISCUSSION THREAD

**Discussion by oldfart101** [Lissy93/dashy#407] [General]
Having an issue with icons on Unraid docker - can't load them!
Unraid settings:
Config file: /mnt/user/appdata/dashy/conf.yml
Icons Path: /mnt/user/appdata/dashy/icons/dashboard-icons/
Dashy icon - icons/dashboard-icons/png/xyz.png

what is the correct path?

**Comment by oldfart101** [comment]
Solved
changed Unraid icons path to: /mnt/user/appdata/dashy/icons/dashboard-icons/png/
and then just the icon name in dashy : xyz.png

**Reply by lissy93** [reply]
Glad you've got it working, and thanks for posting the solution :)

I'm not too familiar with Unraid, it's on my list of things to learn about.
If anyone here who's familiar with Unraid has a  spare moment, it would be really helpful to get a short Unraid deployment guide pushed to the [Deployment Docs](https://github.com/Lissy93/dashy/blob/master/docs/deployment.md#unraid), to help others in the future

**Comment by oldfart101** [comment]
Will have a look tomorrow to see if I can do a 'how to'

**Reply by oldfart101** [reply]
I have a 'how I installed dashy on Unraid' in both pdf format and odt format - can you email me a contact to send the files to?

**Reply by lissy93** [reply]
This would be awesome, thank you so much :) Sure, send it over to `alicia@omg.lol`

**Reply by Tharic99** [reply]
> I have a 'how I installed dashy on Unraid' in both pdf format and odt format - can you email me a contact to send the files to?

@oldfart101 Did this ever get sent over? I realize @Lissy93 is extremely busy, but if you still have the docs available I'm happy to update the guide and push a git change to the [Deployment Docs](https://github.com/Lissy93/dashy/blob/master/docs/deployment.md#unraid)
